src/autolabel/tasks/base.py

```diff
@@ -67,10 +69,10 @@ def __init__(self, config: AutolabelConfig) -> None:
         self._prompt_schema_init()
 
     def _prompt_schema_init(self) -> None:
-        self.use_llama_prompt_schema = self.config.provider() in [
-            ModelProvider.REFUEL,
-            ModelProvider.TGI,
-        ]
+        self.use_llama_prompt_schema = (
+            self.config.provider() == ModelProvider.REFUEL
+            and self.config.model_name() == REFUEL_LLM_MODEL
+        ) or self.config.provider() == ModelProvider.TGI
```

Comment:
We currently concatenate options in the classification task differently based on the model provider as well. We might need a similar change there too.

Reply:
thanks fixing this